plugins/azure/postgresqlserver/flexibleServerPrivateDns.spec.js

```diff
@@ -0,0 +1,96 @@
+var expect = require('chai').expect;
+var flexibleServerPrivateDns = require('./flexibleServerPrivateDns');
+
+const listPostgresFlexibleServer = [
+{
+    "id": "/subscriptions/12345/resourceGroups/Default/providers/Microsoft.DBforPostgreSQL/flexibleServers/test-server",
+    "type": "Microsoft.DBforPostgreSQL/flexibleServers",
+    "storageProfile": {
+        "storageMB": 5120,
+        "backupRetentionDays": 7,
+        "geoRedundantBackup": "Disabled",
+        "storageAutogrow": "Disabled"
+    },
+    "network":{
+        "publicNetworkAccess": "Enabled"
+    }
+},
+{
+    "id": "/subscriptions/12345/resourceGroups/Default/providers/Microsoft.DBforPostgreSQL/flexibleServers/test-server2",
+    "type": "Microsoft.DBforPostgreSQL/flexibleServers",
+    "storageProfile": {
+        "storageMB": 5120,
+        "backupRetentionDays": 7,
+        "geoRedundantBackup": "Disabled",
+        "storageAutogrow": "Disabled"
+    },
+    "network":{
+        "delegatedSubnetResourceId" : "/subscriptions/26a1a07e-06dd-4892-92c9-e4996b0fc546/resourceGroups/cloudsploit-dev/providers/Microsoft.Network/virtualNetworks/omer-virtual-network-test/subnets/default",
```

Comment:
```suggestion
        "delegatedSubnetResourceId" : "/subscriptions/26a1a07e-06dd-4892-92c9-e4996b0fc546/resourceGroups/cloudsploit-dev/providers/Microsoft.Network/virtualNetworks/omer-virtual-network-test/subnets/default",
```
change subscription id to a random value